src/forms/StripePaymentForm/StripePaymentForm.css

```diff
@@ -42,52 +42,103 @@
 }
 
 .paymentHeading {
-  margin: 0 0 14px 0;
+  margin: 0 0 12px 0;
   color: var(--matterColorAnti);
 
-  padding-top: 8px;
-  padding-bottom: 0px;
+  padding-top: 4px;
+  padding-bottom: 2px;
+
+  @media (--viewportMedium) {
+    margin: 0 0 16px 0;
+    padding-top: 4px;
+    padding-bottom: 4px;
+  }
+}
+
+.paymentMethodSelector {
+  margin-bottom: 36px;
+
+  @media (--viewportMedium) {
+    margin-bottom: 40px;
+  }
+}
+
+.saveForLaterUse {
+  padding-top: 6px;
+  margin-bottom: 36px;
+
+  @media (--viewportMedium) {
+    padding-top: 8px;
+    margin-top: 0px;
+    margin-bottom: 48px;
+  }
+}
+
+.saveForLaterUseCheckbox {
+  display: inline-block;
+  padding: 2px 0 3px 0;
+
+  @media (--viewportMedium) {
+    padding: 1px 0 0px 0;
+  }
+}
+
+.saveForLaterUseLabel {
+  @apply --marketplaceH5FontStyles;
+  color: var(--matterColor);
+  margin-top: 3px;
+  margin-bottom: 0px;
+  cursor: pointer;
+}
+
+.saveForLaterUseLegalInfo {
+  @apply --marketplaceH5FontStyles;
+  color: var(--matterColorAnti);
+  display: inline-block;
+  padding: 0px 0 0px 26px;
+  margin-top: -2px;
```

Comment:
I assume there is a good reason to use a negative margin here but I would like to know why as it's usually something we try to avoid 👀

Reply:
@OtterleyW Well, usually these are fixable with more consistent design. 

This time adding an extra vertical space (worth one baseline) to avoid negative margin looked bad and adjusting line-height of our fonts sounded bad too (because custom font-styling per component makes it harder to change fonts - but that's also the case with negative margins. So, I chose between two evils).